aip/0132.md

```diff
@@ -80,7 +81,7 @@ message ListBooksRequest {
   - The comment above the message **should** document the maximum allowed
     value, as well as the default value if the field is omitted (or set to
     `0`). If preferred, the API **may** state that the server will use a
-    sensible default that may vary over time.
+    sensible default. This default **may** change over time.
   - If a user provides a value greater than the maximum allowed value, the API
     **should** coerce the value to the maximum allowed.
   - If a user provides a negative value, the API **must** send an
```

Comment:
I can't comment on the relevant line, but I prefer "relevant" over "germane".

Reply:
Done.